src/shared/UrlContext.ts

```diff
@@ -6,12 +6,19 @@ import type {
 } from './types/UrlContext';
 import { UrlContextParam } from './types/UrlContext';
 
+function getWindowType(params: URLSearchParams): WindowType {
+  if (window.location.pathname.startsWith('/sidepanel')) {
```

Comment:
Do we use this case somewhere?

Reply:
yes, we use `urlContext.windowType` in a lot of places :)